<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>459</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SimSun</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tela principal</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(101, 252, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labeltext0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
    <property name="lineWidth">
     <number>15</number>
    </property>
    <property name="midLineWidth">
     <number>15</number>
    </property>
    <property name="text">
     <string>WEATHER FORECAST</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
    <property name="indent">
     <number>7</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="powerlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POWERED BY: RAFAEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(250, 216, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;
border-color: black;
}

QPushButton:pressed {
background-color: rgb(250, 216, 255);
border-style: outset; 
border-width: 2px;
border-radius: 15px; 
border-color: white;
}</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(250, 216, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;
border-color: black;
}
QPushButton:pressed {
background-color: rgb(250, 216, 255);
border-style: outset; 
border-width: 2px;
border-radius: 15px; 
border-color: white;

}</string>
    </property>
    <property name="text">
     <string>CONFIRMAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>c2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fah2.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(250, 216, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;
border-color: black;
}
QPushButton:pressed {
background-color: rgb(250, 216, 255);
border-style: outset; 
border-width: 2px;
border-radius: 15px; 
border-color: white;

}</string>
    </property>
    <property name="text">
     <string>CONFIRMAR</string>
    </property>
   </widget>
   <zorder>labeltext0</zorder>
   <zorder>powerlabel</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
